test/src/examples_complete_test.go

```diff
@@ -245,9 +248,124 @@ func TestExamplesCompleteWithReplication(t *testing.T) {
 
 }
 
+func TestExamplesCompleteWithPrivilegedPrincipals(t *testing.T) {
+	t.Parallel()
+	rand.Seed(time.Now().UnixNano() + 4)
```

Comment:
The idea is that if, somehow, all the parallel executions got the same Nano Time, they would still get unique seeds by adding a distinct small number likely to be smaller than the number of nanoseconds in a clock tick. 
```suggestion
	rand.Seed(time.Now().UnixNano() + 5)
```